popsycle/synthetic.py

```diff
@@ -299,27 +338,30 @@ def perform_pop_syn(ebf_file, output_root, iso_dir,
     ##########
     # Extend the edges a bit, that's what the * 1.1 is for (to potentially catch any edge cases.)
     # make bins of size ~1/2 arcmin
-    radius = np.sqrt(surveyArea/np.pi)  # degrees
+    radius = np.sqrt(surveyArea / np.pi)  # degrees
 
     # Define bin_edges_number, if not given in input.
     if bin_edges_number is None:
-        bin_edges_number = int(60 * 2 * radius) + 1   # st the widths are 1/2 arcmin
+        bin_edges_number = int(
```

Comment:
minor thing... I am not a fan of this line break. Move the comment up or down; but leave the equation on the same line.

Reply:
fixed